### ISSUE TITLE
google.protobuf.Empty: TypeError: No positional arguments allowed

### ISSUE THREAD

**Issue by apemost** [googleapis/gapic-generator-python#205]
```
    def list_categories(self,
            request: empty.Empty = None, *,
            retry: retries.Retry = gapic_v1.method.DEFAULT,
            timeout: float = None,
            metadata: Sequence[Tuple[str, str]] = (),
            ) -> sth.ListCategoriesResponse:
        r"""Lists categories.

        Args:
            request (:class:`~.empty.Empty`):
                The request object. A generic empty message that you can
                re-use to avoid defining duplicated empty messages in
                your APIs. A typical example is to use it as the request
                or the response type of an API method. For instance:

                    service Foo {
                      rpc Bar(google.protobuf.Empty) returns
                (google.protobuf.Empty);     }

                The JSON representation for `Empty` is empty JSON object
                `{}`.
            retry (google.api_core.retry.Retry): Designation of what errors, if any,
                should be retried.
            timeout (float): The timeout for this request.
            metadata (Sequence[Tuple[str, str]]): Strings which should be
                sent along with the request as metadata.

        Returns:
            ~.sth.ListCategoriesResponse:
                Response for listing categories.
        """
        # Create or coerce a protobuf request object.
>       request = empty.Empty(request)
E       TypeError: No positional arguments allowed

**Comment by software-dov** [comment]
What input did you feed in? What python version are you running? Give us some details, please.
http://catb.org/~esr/faqs/smart-questions.html

**Comment by software-dov** [comment]
Tentatively fixed by #317
